Userland/Libraries/LibWeb/CSS/StyleProperties.cpp

```diff
@@ -191,23 +178,63 @@ void StyleProperties::load_font(Layout::Node const& node) const
         }
     }
 
-    FontSelector font_selector { family, size, weight };
+    // Note: Both of these are modified by the find_font() lambda
+    FontSelector font_selector;
+    bool monospace = false;
 
-    auto found_font = FontCache::the().get(font_selector);
-    if (found_font) {
-        m_font = found_font;
-        return;
-    }
+    auto find_font = [&](String const& family, bool is_monospace = false) -> RefPtr<Gfx::Font> {
+        font_selector = { family, size, weight };
+        monospace = is_monospace;
```

Comment:
I don't like the way `font_selector` and `monospace` are overwritten. I tried using a local struct and returning it from the two lambdas above, but it didn't work as well as I'd hoped:
```cpp
struct FoundFont {
    FontSelector selector;
    bool monospace;
    NonnullRefPtr<Gfx::Font> font;
};
```

Because then, when you get to line 238, we don't know the value of monospace so it was pointless. Maybe just writing to the variables is the best option?